### PR TITLE
book-content structure setted

### PR DESCRIPTION
**Pull request by renzowolox**
## Summary
- book content structure setted
- shadow added to this box
- book title underlined

## Screenshots
image

## Trello Card
https://trello.com/c/0dXQ9N83/28-detalle-de-libro-iii-estructura#

### CODE REVIEW

File: maquetado/index.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
   <meta name="author" content="Wolox">
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
-<body>
+<body class="main-content">
```
Comment (leonardofrp5) [inline maquetado/index.html]: the tag body can has class

File: maquetado/scss/detail.scss [hunk 2]
```diff
@@ -1,8 +1,17 @@
 @import 'colors';
 
+.main-content {
+  display: flex;
+  justify-content: center;
+  padding-top: 20px;
+}
+
 .book-content {
+  border: 1px solid $alabaster;
```
Comment (leonardofrp5) [inline maquetado/scss/detail.scss]: I can this delete this line

File: maquetado/index.html (hunk 1 above)
Comment (renzowolox) [reply maquetado/index.html]: I just added a new div with this class

File: maquetado/scss/detail.scss (hunk 2 above)
Comment (renzowolox) [reply maquetado/scss/detail.scss]: yes, the line has been removed

File: maquetado/index.html [hunk 3]
```diff
@@ -8,16 +8,18 @@
...
-      <img src="images/book-cover.png" height="350" width="250" alt="book-image">
-    </div>
-    <div class="book-text">
-      <h3 class="book-title">Título del libro <span class="text-1">(género)</span></h3>
-      <p class="book-detail">Autor del libro: <span class="text-2">Nombre del autor del libro</span></p>
-      <p class="book-detail">Editorial: <span class="text-2">Nombre de la editorial</span></p>
-      <p class="book-detail">Año de publicación: <span class="text-2">Año de publicación</span></p>
-    </div>
+  <div class="main-content">
+    <div class="book-content">
```
Comment (leonardofrp5) [inline maquetado/index.html]: why tow div ?, I can use only one

File: maquetado/scss/detail.scss [hunk 4]
```diff
@@ -1,8 +1,16 @@
...
+  display: flex;
+  justify-content: center;
+  padding-top: 20px;
+}
+
 .book-content {
+  box-shadow: 4px 4px 7px grey;
   display: flex;
   padding: 30px 25px;  
+  width: 70%;
```
Comment (Dorian30) [inline maquetado/scss/detail.scss]: We generally don't use relative sizes 'cause they tend to be unpredictable. You can find all the sizes you need in the invision. For responsive purposes we use breakpoints. For now we can do something like 

```css
max-width: some size in pixels;
width: 100%;

File: maquetado/scss/detail.scss [hunk 5]
```diff
@@ -12,12 +20,15 @@
 .book-text {
   display: flex;
   flex-direction: column;
+  flex-grow: 1;
```
Comment (Dorian30) [inline maquetado/scss/detail.scss]: I think you don't need this here 👀

File: maquetado/scss/detail.scss (hunk 4 above)
Comment (leonardofrp5) [reply maquetado/scss/detail.scss]: correct

File: maquetado/scss/detail.scss (hunk 5 above)
Comment (renzowolox) [reply maquetado/scss/detail.scss]: i added this property for match the design with the card image
- without flex-grow
image

File: maquetado/scss/detail.scss [hunk 6]
```diff
@@ -12,12 +20,15 @@
...
 }
 
 .book-title {
+  border-bottom: 3px solid $pear;
   font-size: 28px;
   font-weight: bold;
   margin-bottom: 50px;
+  padding-bottom: 10px;
```
Comment (Dorian30) [inline maquetado/scss/detail.scss]: You are already using margin bottom here. Why you need to add padding bottom ?

File: maquetado/scss/detail.scss (hunk 5 above)
Comment (Dorian30) [reply maquetado/scss/detail.scss]: Perfect! I hadn't notice you had applied this to the whole info container. Good work there.

File: maquetado/scss/detail.scss (hunk 6 above)
Comment (renzowolox) [reply maquetado/scss/detail.scss]: I'm using padding bottom to separate the title from underline and the margin is used to separate the title from details.
image

Comment (Dorian30) [reply maquetado/scss/detail.scss]: Awesome!

**Pull request merged**